pkg/provider/kubernetes/crd/kubernetes_http.go

```diff
@@ -409,6 +409,40 @@ func (c configBuilder) loadServers(parentNamespace string, svc traefikv1alpha1.L
 		}), nil
 	}
 
+	if svc.NodePortLB && service.Spec.Type == corev1.ServiceTypeNodePort {
+		nodes, nodesExists, nodesErr := c.client.GetNodes()
+		if nodesErr != nil {
+			return nil, nodesErr
+		}
+
+		if !nodesExists || len(nodes) == 0 {
+			return nil, fmt.Errorf("nodes not found in namespace %s", namespace)
```

Comment:
```suggestion
			return nil, fmt.Errorf("nodes not found in namespace %q", namespace)
```

Reply:
Is there a convention for this? I see mixed usage of %s and %q throughout the code.

Reply:
I see %s being used more in namespace printing. So please confirm that you prefer "%q" here.

Reply:
No I don't think there's a convention on that, perhaps a maintainer can confirm.
But rereading the message, I think we don't care, because, the nodes are not namespaced. 
So I will update my suggestion to reflect that.

Reply:
@youkoulayley is right, we have no strong convention for this, and as you mentioned we used to use both.

Reply:
You ordered the arguments wrong, but I use your updated text now